README.md

```diff
@@ -35,38 +35,36 @@ We'll implement the tool in several incremental phases to get some basic functio
  
 Based on Tim's feedback, instead of using `<xsdl:include/>` as originally proposed, change it so that it can be used to either include _all_ the scripts in the referenced file:
 
-	<Binding>
-		<Definition>
-			<Script>
-				<Transforms>
-					<xsdl:EspRequest>
-						<xsdl:include-from file="service-scripts.xml" all="true"/>
-						<!-- 
-								All the transforms defined in service-scripts.xml
-								are included here in this element
-						-->
-					</xsdl:EspRequest>
-				</Transforms>
-			</Script>
-		</Definition>	
-	</Binding>
+  <Binding>
+    <Definition>
+      <Script>
+        <Transforms>
+          <EspRequest>
+            <include file="service-scripts.xml"/>
```

Comment:
Removed the xsdl prefix.  I don't think this is an urn:hpcc:esdl:script command.  It's to pull scripts into the entry point.  I think the entrypoint can contain top level script (transform or transforms) or include statements to load transforms from a file.
It's a very picky because i think there will also be xsdl:include in the scripting language, and it will be basically the same, but I just think this is a framework command, processed as part of the framework not by evoking a sciprt.

Reply:
Ok yes that makes sense. I wasn't paying close enough attention to the prefixes.

Reply:
I'm ok with the change, but I think the distinction between include and xsdl:include needs more discussion. Obviously a configuration split into multiple pieces requires a way to identify and locate those pieces, but is that script syntax or framework overhead? My module concept gets pushback for being too complicated, but my initial thoughts about transforms including transforms makes me think instead of a relatively simple module tree we're going to be looking at a completely random module graph (each included file would represent a module).

My concept of a transform is just a variable scope that may optionally update the XPath context root(s). This is why an entry-point is just a special case of a transform, the same as a called function or applied template. There's no reason one inline transform couldn't include another, except to limit the complexity introduced by arbitrary cross references.

Reply:
When you say "...is that script syntax or framework overhead" are you saying that the 'binding build tool' would not be responsible for walking the script files, compiling all the includes and generating a single Binding Config File? That instead the script engine runtime would do that work?

Reply:
I don't know. That's the question I'm asking. Tony mentioned two distinct includes, one for the binding build tool and one in markup. I want more discussion about the perceived distinction. That discussion is not a prerequisite for accepting this PR, but its needed.